### PR TITLE
Make docs easier to run locally

### PR DESCRIPTION
**Pull request by fao89**
[noissue]

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -45,9 +46,17 @@ clean:
 	rm -rf $(BUILDDIR)/*
 	rm -rf $(DIAGRAM_BUILD_DIR)/*
 
+install:
+	python3 -m venv pulpdocs
+	source pulpdocs/bin/activate && pip install -r ../doc_requirements.txt
```
Comment (melcorr) [inline docs/Makefile:51]: @fao89 so if I run this, I should be able to build?

**Comment by fao89** [comment]
@melcorr my idea was having everything under makefile to make easier to build docs.

make install - you install all the requirements
make dirhtml - you build the docs (may need to run make diagrams first)
make run - you start a server to get the docs running

in the first-run you can combine all the make commands and just do:
```
make install diagrams dirhtml run
```

FYI @bmbouter @ipanova

File: docs/Makefile (hunk 1 above)
Comment (fao89) [reply docs/Makefile:51]: you replied before I finished my comment below, let me know if my comment was helpful or not

**Comment by melcorr** [comment]
Thanks @fao89 - I'll test later today. Thank you.

File: docs/Makefile (hunk 1 above)
Comment (melcorr) [reply docs/Makefile:51]: It was so helpful I thought it was a direct response to ^

**Review by melcorr — APPROVED** [review]
I tested this. It is a dream. 
I needed to run: sudo dnf install python3-devel 
It's a prereq. It works now seamlessly. 
Thanks so much @fao89

**Review by bmbouter — APPROVED** [review]
I didn't run this because my dev env already has these things, but +1 to it and I think it's right.

**Pull request merged**